R/InstanceAPI.R

```diff
@@ -19,21 +19,16 @@ InstanceAPI <- R6::R6Class( # nolint object_name_linter
     #' Get the schema for the instance.
     get_schema = function() {
       # TODO: replace with laminr.api get_schema call
-      request <- httr::GET(
-        paste0(
-          private$.instance_settings$api_url,
-          "/instances/",
-          private$.instance_settings$id,
-          "/schema"
-        )
+      url <- paste0(
+        private$.instance_settings$api_url,
+        "/instances/",
+        private$.instance_settings$id,
+        "/schema"
       )
 
-      content <- httr::content(request)
-      if (httr::http_error(request)) {
-        cli_abort(content$detail)
-      }
+      request <- httr::GET(url)
```

Comment:
Should this be "response" rather than "request"?

Reply:
good point